MetaMask/metamask-extension
# Replace useSafeGasEstimatePolling with usePolling

## **Description**
Added `usePolling` hook and used it within `useSafeGasEstimatePolling` hook.

This replaces where gas estimates are coming from for the confirmation flow.

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2014

## **Manual testing steps**

1. Hit send button
2. get to confirmation screen
3. see that gas estimation updates

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
